README.md

```diff
@@ -9,6 +9,7 @@
 - Kitty Items is a **complete NFT marketplace** built with [Cadence](https://docs.onflow.org/cadence), Flow's resource-oriented smart contract programming language.
 - Learn how to deploy contracts, mint NFTs, and integrate user wallets with the Flow Client Library (FCL).
 
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```

Comment:
I tried out this button and ran into this error. Maybe we should create a new section "Deploy with Heroku" that walks through the extra steps needed to make it work?

Also I'm not sure why it's saying invalid `app.json`, though I wonder if it's because it's trying to find `app.json` on the Kitty Items `master` branch, which doesn't have it yet

<img width="676" alt="Screen Shot 2021-04-04 at 12 37 16 PM" src="https://user-images.githubusercontent.com/2547035/113519999-e0659300-9544-11eb-84c5-29f7748d6937.png">

Reply:
The user will have to fill oiut the env vars in app.json for the one button deploy to work. I'll cretwe a seperate `.md` file that fgives instructions for the Heroku deployment process.

Reply:
This should be fixed now.